ui/razercfg

```diff
@@ -387,6 +414,7 @@ def usage():
 	print("-R|--getres                         Prints the resolutions")
 	print("-f|--freq [PROF:]FREQ               Changes the scan frequency")
 	print("-F|--getfreq                        Prints the frequencies")
+	print("-n|--mapping [PROF:]RES[xRES]       Changes the resolution of the currently used mapping")
```

Comment:
Yes, you are absolutely right about upper/lower case.
But -m is already used for LED mode.
What about -a?